File: valid-anagram/valid-anagram.go
func isAnagram(s string, t string) bool {
    if len(s) != len(t){
        return false
    }
    
    letter := map[byte]int{}
    for i:= 0; i<len(s); i++{
        letter[s[i]]++
        letter[t[i]]--
    }
    
    for _, v := range letter{
        if v != 0{
            return false
        }
    }
    return true
}

File: valid-anagram/valid-anagram.py
class Solution:
    def isAnagram(self, s: str, t: str) -> bool:
        count = [0]*26
        if len(s) != len(t):
            return False
        for i in range(len(s)):
            count[ord(s[i]) - ord('a')] += 1
            count[ord(t[i]) - ord('a')] -= 1
        
        for i in range(26):
            if count[i] != 0:
                return False
        return True